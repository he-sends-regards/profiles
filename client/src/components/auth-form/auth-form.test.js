import React from 'react';
import renderer from 'react-test-renderer';
import AuthForm from './auth-form.jsx';

it('renders correctly', () => {
  const tree = renderer
      .create(<AuthForm />)
      .toJSON();
  expect(tree).toMatchSnapshot();
});
