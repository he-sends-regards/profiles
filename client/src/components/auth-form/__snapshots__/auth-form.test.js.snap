// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="auth-form"
>
  <div
    aria-label="Basic example"
    className="auth-form__auth-buttons btn-group"
    role="group"
  >
    <button
      className="btn btn-primary"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      Login
    </button>
    <button
      className="btn btn-primary"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      Register
    </button>
  </div>
  
</div>
`;
